store/storetest/team_store.go

```diff
@@ -387,9 +395,7 @@ func testTeamStoreGetByInviteId(t *testing.T, ss store.Store) {
 	o1.Type = model.TEAM_OPEN
 	o1.InviteId = model.NewId()
 
-	save1 := <-ss.Team().Save(&o1)
-
-	if err := save1.Err; err != nil {
+	if _, err := ss.Team().Save(&o1); err != nil {
```

Comment:
You are changing here the behavior, the expect thing is you Save the o1 object and get the save1 and err (because the save1.InviteId is used some lines after.